.github/workflows/build-and-test-differential.yaml

```
@@ -11,14 +11,10 @@ jobs:
       fail-fast: false
       matrix:
         rosdistro:
-          - galactic
           - humble
         container-suffix:
           - -cuda
         include:
-          - rosdistro: galactic
-            container: ghcr.io/autowarefoundation/autoware-universe:galactic-latest
-            build-depends-repos: build_depends.repos
           - rosdistro: humble
             container: ghcr.io/autowarefoundation/autoware-universe:humble-latest
             build-depends-repos: build_depends.humble.repos
```

```suggestion
            build-depends-repos: build_depends.repos
```

I think this should be like this because thir PR is removing the `build_depends.humble.repos`

---

Maybe already done in 
https://github.com/autowarefoundation/autoware.universe/pull/2408/commits/f8653750a3d75bf7af04d03fb1013909ffef60f6?

---

I'm sorry my work was late, I forgot about that and noticed just before. :pleading_face:

---

Ah yes, when I was checking it wasn't there :D